src/animations/cssTransitions.ts

```diff
@@ -0,0 +1,83 @@
+let browserSpecificTransitionEndEventName = '';
+let browserSpecificAnimationEndEventName = '';
+
+interface VNodeProperties {
+	enterAnimationActive?: string;
+	exitAnimationActive?: string;
+}
+
+function determineBrowserStyleNames(element: HTMLElement) {
+	if ('WebkitTransition' in element.style) {
+		browserSpecificTransitionEndEventName = 'webkitTransitionEnd';
+		browserSpecificAnimationEndEventName = 'webkitAnimationEnd';
+	}
+	else if ('transition' in element.style) {
```

Comment:
That makes sense 👍 This was from the original maquette code and I guess I didn't optimize it. Fixed!